### DISCUSSION TITLE
Increment to 0.4.1?

### DISCUSSION THREAD

**Discussion by rijobro** [Project-MONAI/MONAI#1477] [General]
I'm running some of the notebooks which download data in colab. 0.4.0 doesn't have [this PR](https://github.com/Project-MONAI/MONAI/pull/1450), so it's very slow. Any chance we could tag 0.4.1? 

@wyli @Nic-Ma @ericspod

**Comment by wyli** [comment]
I think in this case we should use the weekly build for a specific feature https://pypi.org/project/monai-weekly/ for some reason the weekly build is out of date at the moment
